docs/user-manual/modules/ROOT/pages/camel-4-migration-guide.adoc

```diff
@@ -16,35 +16,39 @@ Camel 4 supports Java 17. Support for Java 11 is dropped.
+|===
+| Component | Reason | Alternative component(s)                                           
+| camel-activemq               |        | camel-jms / camel-sjms / camel-amqp                        
+| camel-any23                  |        |                                                        
+| camel-atmos                  |        |                                                        
+| camel-caffeine-lrucache      |        |                                                        
+| camel-cdi                    |        |                                                        
+| camel-elasticsearch-rest     |        | elasticsearch-component                                
+| camel-gora                   |        |                                                        
+| camel-hbase                  |        |                                                        
+| camel-iota                   |        |                                                        
+| camel-ipfs                   |        |                                                        
+| camel-jbpm                   |        |                                                        
+| camel-jclouds                |        |                                                        
+| camel-johnzon                |        | jackson-dataformat / fastjson-dataformat / gson-dataformat 
+| camel-microprofile-metrics   |        |                                                        
+| camel-milo                   |        |                                                        
+| camel-opentracing            |        |                                                        
+| camel-optaplanner            |        |                                                        
+| camel-rabbitmq               |        | spring-rabbitmq-component                              
+| camel-rest-swagger           |        | openapi-rest                                           
+| camel-restdsl-swagger-plugin |        |                                                        
+| camel-resteasy               |        | camel-rest                                             
+| camel-spark                  |        |                                                        
+| camel-spring-integration     |        |                                                        
+| camel-swift                  |        |                                                        
+| camel-swagger-java           |        | openapi-java                                           
+| camel-websocket              |        | vertx-websocket                                        
+| camel-websocket-jsr356       |        | vertx-websocket                                        
+| camel-xstream                |        | jacksonXml-dataformat                                  
+| camel-zipkin                 |        |                                                        
+|===
```

Comment:
microprofile-metrics => open-telemetry
zipkin -> open-telemtry
opentracing -> open-telemtry

Reply:
I think we removed `camel-vertx-kafka` as well. And we should point it to `camel-kafka`.